adapted-articles/C/Crown.xml

```diff
@@ -2,10 +2,10 @@
 <p>The word “crown” is used metaphorically to indicate rule or royalty (<a href="?bref=Nah.3.17">Na 3:17</a>, <span class="sc">kjv</span>), glory or honor (<a href="?bref=Job.19.9">Jb 19:9</a>; <a href="?bref=Ps.8.5">Ps 8:5</a>; <a href="?bref=Ezek.16.12">Ez 16:12</a>), joy (<a href="?bref=Ezek.23.42">Ez 23:42</a>), or pride (<a href="?bref=Job.31.36">Jb 31:36</a>; <a href="?bref=Isa.28.3">Is 28:3</a>).</p>
-<p>In the NT the most common word for “crown” means a laurel wreath worn at banquets or a prize given as a civic or military honor. The apostle Paul alluded to its use as an athletic prize when he urged Christians to be disciplined in striving for a “crown” that would not wither (<a href="?bref=1Cor.9.25">1 Cor 9:25</a>; <a href="?bref=2Tim.2.5">2 Tm 2:5</a>). Paul regarded his converts as his “joy and crown” (<a href="?bref=Phil.4.1">Phil 4:1</a>; <a href="?bref=1Thes.2.19">1 Thes 2:19</a>).</p>
-<p>A victor’s wreath symbolizes the eternal life inherited by Christians who have persevered (<a href="?bref=Jas.1.12">Jas 1:12</a>; <a href="?bref=1Pet.5.4">1 Pt 5:4</a>; <a href="?bref=Rev.2.10">Rv 2:10</a>; <a href="?bref=Rev.3.11">3:11</a>). In the book of Revelation the victories of the locusts (<a href="?bref=Rev.9.7">9:7</a>), the woman (<a href="?bref=Rev.12.1">12:1</a>), and Christ (<a href="?bref=Rev.6.2">6:2</a>; <a href="?bref=Rev.14.14">14:14</a>) are symbolized by laurel crowns. A different Greek word, meaning a royal crown, is used for the diadems on the heads of the dragon (<a href="?bref=Rev.12.3">12:3</a>), the beast from the sea (<a href="?bref=Rev.13.1">13:1</a>), and Christ (<a href="?bref=Rev.19.12">19:12</a>).</p>
-<p>Jesus’ crown of thorns was a circlet formed from a prickly shrub—an ironic parody of a victor’s wreath (<a href="?bref=Mark.15.17-18">Mk 15:17-18</a>). Its combination with the robe, scepter (<a href="?bref=Matt.27.27-29">Mt 27:27-29</a>), and satirical inscription on the cross that Jesus was “the King of the Jews” (<a href="?bref=Mark.15.26">Mk 15:26</a>), were all meant to mock him as a defeated messianic aspirant.</p>
+<p>In the NT the most common word for “crown” means a laurel crown worn at banquets or a prize given as a civic or military honor. The apostle Paul used it as an athletic prize when he urged Christians to strive for a “crown” that would not weaken (<a href="?bref=1Cor.9.25">1 Cor 9:25</a>; <a href="?bref=2Tim.2.5">2 Tm 2:5</a>). Paul regarded his converts as his “joy and crown” (<a href="?bref=Phil.4.1">Phil 4:1</a>; <a href="?bref=1Thes.2.19">1 Thes 2:19</a>).</p>
```

Comment:
- Not sure "Paul used it as an atheltic prize" has the same meaning as "alluded to its use as an athletic prize".

Reply:
- Wither I= weaken? Wither = se gâter, se flétrir